.github/workflows/ci.yml

```diff
@@ -110,6 +110,12 @@ jobs:
             echo “Nothing to test”
           else
             pytest $args
+            ret=$?
+            echo $ret
+            if [ "$ret" = 5 ]; then
+              echo "No tests collected."
+              exit 0
+            fi
```

Comment:
curious: how did you run into this?

Reply:
good question :D not all sample projects have python code/python tests. by default, pytest exits with error code 5 if that's the case. 
The action is still not passing though still so I gotta look into it more (it worked when I tested locally).

Reply:
I see, the workflow is failing. I'm a bit rusty, does GH pick up the new workflow from the PR, or would you have to merge this first? If it's running the version from `main`, your fix won't take effect yet...

Reply:
Also, the CI is known to be broken right now. So no worries about failing runs at the moment, just run the tests locally and confirm they're passing before merge.